packages/markdown-docx/src/ToOOXMLVisitor/index.js

```diff
@@ -86,21 +94,40 @@ class ToOOXMLVisitor {
                     let textValueTag = TEXT_RULE(subNode.text);
 
                     let tag = TEXT_WRAPPER_RULE(propertyTag, textValueTag);
+
+                    if (isLinkPropertyPresent) {
+                        // some rels are already present
+                        // To avoid overwrite ids start our
+                        // rels from 10
+                        let relationShipId = 'rId' + (this.relationships.length + 10).toString();
```

Comment:
2 is sufficient, I think. We only have two relationship tags as of now.

Reply:
`relationshipId`. Please change it everywhere.

Reply:
@algomaster99 I am unsure about 2. I would at least keep it 6/7 as word default enters 5-6 relationship Ids.

Reply:
@algomaster99 Currenlty, I went with 2 in the latest commit.